apps/web/src/components/CommunityHolderGrid/CommunityHolderGridItem.tsx

```diff
@@ -94,21 +89,27 @@ export default function CommunityHolderGridItem({ holderRef, queryRef }: Props)
       </StyledGalleryLink>
```

Comment:
Still remain to render as `image` for all the tokens. Tried to replace with live render and seems like its consume too much memory. Especially for svg token

Reply:
Updated it to use `NftPreview` component